src/app/components/elements/list-groups/ListGroupFooter.tsx

```diff
@@ -10,48 +10,44 @@ interface FooterLinkProps {
 const FooterLink = ({ linkTo, children }: FooterLinkProps) => {
   return (
     <ListGroupItem className="border-0 px-0 py-0 bg-transparent align-items-stretch">
-      <Link className="footerLink py-2" to={linkTo}>
+      <Link className="footerLink" to={linkTo}>
         {children}
       </Link>
     </ListGroupItem>
   );
 };
 
-let key = 0;
 const footerLinks = {
   left: [
-    <FooterLink key={key++} linkTo="/about">
+    <FooterLink key="about-us-footer-link" linkTo="/about">
       About us
     </FooterLink>,
-    <FooterLink key={key++} linkTo="/contact">
+    <FooterLink key="contact-us-footer-link" linkTo="/contact">
       Contact us
     </FooterLink>,
-    <FooterLink key={key++} linkTo="/accessibility">
-      Accessibility <span className="d-none d-md-inline">statement</span>
-    </FooterLink>,
-    <FooterLink key={key++} linkTo="/safeguarding">
+    <FooterLink key="safeguarding-footer-link" linkTo="/safeguarding">
       Safeguarding
     </FooterLink>,
-  ],
-  right: [
-    <FooterLink key={key++} linkTo="/privacy">
-      Privacy policy
-    </FooterLink>,
-    <FooterLink key={key++} linkTo="/terms">
-      Terms of use
+    <FooterLink key="student-support-footer-link" linkTo="/support/student">
+      Student Support
     </FooterLink>,
-    <FooterLink key={key++} linkTo="/cookies">
-      Cookie policy
+    <FooterLink key="teacher-support-footer-link" linkTo="/support/teacher">
+      Teacher Support
     </FooterLink>,
   ],
 };
 
 export const ListGroupFooter = () => (
   <div className="footer-links">
-    <h2 className="h5">Links</h2>
-    <div className="d-flex flex-row pt-lg-3">
-      <ListGroup className="w-50 mb-3 link-list">{footerLinks.left}</ListGroup>
-      <ListGroup className="w-50 mb-3 link-list">{footerLinks.right}</ListGroup>
+    <div className="footer-support-links d-flex flex-column flex-md-row align-items-start mt-3 pb-4 py-lg-3">
```

Comment:
Likewise here, the vertical padding seems far too large (top) compared to the designs (bottom):

<img width="378" alt="image" src="https://github.com/user-attachments/assets/04efded9-5115-43f3-95fc-6510d7b9b2f3">
<img width="613" alt="image" src="https://github.com/user-attachments/assets/cdbde7a1-63d8-4996-bf8b-f5770c36ed13">

Reply:
@sxpydo - I'm convinced this change has worked. The designs want 20px of horizontal padding and 20px of vertical padding. We now have 15px horizontal and 48px vertical. Can this be double checked one more time please? 

Likewise with the "Links" section on desktop, it's still slightly skewed compared to the other sections on the padding :)